Makefile.am

```diff
@@ -1,6 +1,10 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = transport local owr bridge tests docs bindings
+SUBDIRS = transport local owr tests docs
+
+if OWR_BRIDGE
+SUBDIRS += bridge bindings
```

Comment:
`bindings` shouldn't be included here, we still want those even if we don't build the bridge. However we don't want to build the bridge part of the bindings.